packages/manager/src/components/Button/button.spec.js

```diff
@@ -1,148 +1,166 @@
+      $(button.generic).waitForDisplayed();
+    });
+
+    it('should display dropdown buttons with carat', () => {
+      primaryDropdowns = $$(button.primaryDropdown);
+      primaryDropdowns.forEach(d => {
+        expect(d.$('svg').isDisplayed())
+          .withContext(`svg should be displayed`)
+          .toBe(true);
+      });
+    });
+
+    it('should have primary included in class of each dropdown', () => {
+      primaryDropdowns.forEach(d => {
+        expect(d.getAttribute('class').includes('Primary'))
+          .withContext(`missing Primary class`)
+          .toBe(true);
+      });
+    });
+  });
+
+  describe('Secondary Dropdown', () => {
+    let secondaryDropdowns;
+
+    beforeAll(() => {
+      navigateToStory(component, childStories[3]);
+      $(button.generic).waitForDisplayed();
+    })
+
+    it('should display dropdown buttons with carat', () => {
+      secondaryDropdowns = $$(button.secondaryDropdown);
+      secondaryDropdowns.forEach(s => {
+        expect(s.$('svg').isDisplayed())
+          .withContext(`svg should be displayed`)
+          .toBe(true);
+      });
+    });
+
+    it('should have secondary included in class of each dropdown', () => {
+      secondaryDropdowns.forEach(d => {
+        expect(d.getAttribute('class').includes('Secondary'))
+          .withContext(`missing Secondary class`)
+          .toBe(true);
+      });
+    });
+  });
+
+  describe('Destructive Button', () => {
 
-    xdescribe('Destructive Button', () => {
-
-        beforeAll(() => {
-            navigateToStory(component, childStories[4]);
-        });
-        
-        let destructiveButtons;
-
-        it('should display an enabled destructive button and a disabled button', () => {
-            destructiveButtons = $$(button.destructive);
-            const disabledButtons = destructiveButtons.filter(d => d.getAttribute('disabled') !== null);
-            
-            destructiveButtons.forEach(d => {
-                expect(d.isVisible()).toBe(true);
-            });
-            expect(disabledButtons.length).toBe(1);
-        });
+    beforeAll(() => {
+      navigateToStory(component, childStories[4]);
     });
 
-    xdescribe('Disabled Button', () => {
-        let secondaryDropdowns;
+    let destructiveButtons;
 
-        it('should display one disabled button', () => {
-            disabledDropdowns = secondaryDropdowns.filter(s => s.getAttribute('disabled') !== null);
-            expect(disabledDropdowns.length).toBe(1);
-        });
+    it('should display an enabled destructive button and a disabled button', () => {
+      destructiveButtons = $$(button.generic)
+      const disabledButtons = destructiveButtons.filter(d => d.getAttribute('class').includes('disabled'));
 
+      destructiveButtons.forEach(d => {
+        expect(d.isDisplayed())
+          .withContext(`destructive button should be displayed`)
+          .toBe(true);
+      });
+      expect(disabledButtons.length)
```

Comment:
should we add an assertion for the destructiveButtons length as as well, and an assertion for the disabledButtons displayed-ness?

Reply:
I don't think it hurts either way in this case but I added both assertion types.